_maps/pubbystation.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`The science order console has no valid areas to deliver to on this map, add some more fallback areas to its "department_delivery_areas" var.`
<hr>

I'm not actually sure if my solution to this is the best one, but it does at least work to fix the error.
The error is being caused by the science order console wanting to deliver to `/area/science/research`, but on Pubbystation that area isn't actually used anywhere.
https://github.com/pariahstation/Pariah-Station/blob/434e9eecaf65c0b0b0091b32585ff2054db7811d/code/modules/cargo/department_order.dm#L182-L185
The fix I used was to replace `/area/science/explab` with `/area/science/research`, since that seems to be the largest single area of the science department.
![ezgif com-gif-maker](https://user-images.githubusercontent.com/57483089/169619554-c9241b7c-0ab7-4a61-9e04-7b581ef842fc.gif)